roles/prometheus/templates/etc/systemd/system/prometheus.service.j2

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Prometheus
+After=network.target
+
+[Service]
+Type=simple
+EnvironmentFile=/etc/default/prometheus
+PIDFile=/var/run/prometheus.pid
+ExecReload=/bin/kill -HUP $MAINPID
+ExecStart=/opt/prometheus/current/prometheus-2.0.0-rc.1.linux-amd64/prometheus --config.file=/etc/prometheus/prometheus.yml --web.listen-address=:9090 --web.external-url=demo.robustperception.io --web.console.libraries=/opt/prometheus/current/prometheus-2.0.0-rc.1.linux-amd64/console_libraries --web.console.templates=/opt/prometheus/current/prometheus-2.0.0-rc.1.linux-amd64/consoles
```

Comment:
I'd template out the version numbers here, they appear in a few too many places so it's likely someone would forget to update them. You could also use a symlnk.

Where's the storage directory?